### PR TITLE
Script Libraries blog post

### PR DESCRIPTION
**Pull request by paulbouwer**

### CODE REVIEW

**Comment by adamralph** [comment]
:heart: it!

@glennblock any comments before we merge?

**Comment by glennblock** [comment]
Awesome post Paul! I have a few small edits.

Also, we shouldn't merge till we release 0.14.

**Comment by adamralph** [comment]
I've updated the checklist at https://github.com/scriptcs/scriptcs/issues/947 to reflect that

File: _posts/2015-03-07-script-libraries.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+}
+ 
+public double Subtract(double a, double b) {
+  logger.Info(String.Format("Subtracting {0} - {1}", a, b));
+  return a-b;
+}
+```
+You can see that this script contains basic `Add` and `Subtract` functions. 
+
+It is possible to pull in a traditional Script Pack via the `Require<T>` mechanism. Here we are pulling in the `Logger` Script Pack. Note that you cannot use `var` to declare the variable holding the Script Pack within Script Libraries. You **MUST** use an explictly typed variable. For the background on why, refer to the *Adding a library reference* section of the Script Libraries [Design Document](https://github.com/scriptcs/scriptcs/wiki/Script-Libraries).
```
Comment (jrusbatch) [inline _posts/2015-03-07-script-libraries.md]: We should link directly to the section being referred to here.

**Comment by jrusbatch** [comment]
Looks great! Nice post!

File: _posts/2015-03-07-script-libraries.md (hunk 1 above)
Comment (paulbouwer) [inline _posts/2015-03-07-script-libraries.md]: Done.

**Comment by paulbouwer** [comment]
@glennblock Send through the edits and I'll update the post.

**Comment by glennblock** [comment]
Will do it today

**Comment by adamralph** [comment]
Is this ready to go? The code is ready to be released.

**Comment by adamralph** [comment]
The date will need changing, so whoever does the merge should pull down Paul's branch locally, add a commit to change the date and then push to upstream master. That will also close this PR.

**Comment by paulbouwer** [comment]
It's good to go from my side. I think @glennblock wanted some edits to the
post that still need to be made.

**Pull request merged**